src/lib/hasInvisibleChars.js

```diff
@@ -0,0 +1,13 @@
+import assertString from './util/assertString';
+
+// check this site https://invisible-characters.com/ for list of invisible chars. I have excluded the space characters that are part of \s regex check
+
+// eslint-disable-next-line max-len
```

Comment:
Can you move this down to directly before the declaration of `invisibleChars`?

Reply:
@WikiRik - sure

Reply:
Done